testbed/HIL_Firmware/PDU_HIL/PDU_HIL/pduOutputs.c

```diff
@@ -0,0 +1,48 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "driver/twai.h"
+#include "driver/gpio.h"
+#include "pduOutputs.h"
+
+void (*relayPduPtr)(void*) = &relayPduOutputs;
```

Comment:
not obvious to me but why you want a function pointer over just the function itself?